metrics-facade-prometheus/src/main/java/com/ringcentral/platform/metrics/samples/prometheus/collectorRegistry/SimpleCollectorRegistryPrometheusInstanceSamplesProvider.java

```diff
@@ -77,7 +77,7 @@ public SimpleCollectorRegistryPrometheusInstanceSamplesProvider(
         Collection<? extends CollectorRegistry> collectorRegistries) {
 
         this(
-            n -> MetricName.of(namePrefix, n),
+                namePrefix == null ? MetricName::of : n -> MetricName.of(namePrefix, n),
```

Comment:
Thank you for fixing that! :)
I'd suggest changing indentation for consistency with the other params.

Reply:
fixed,  thank you.